cli/src/commands/schema/edit.ts

```diff
@@ -298,16 +299,22 @@ Beware that this can lead to ${chalk.bold(
   getMessageForColumn(table: EditableTable, column: EditableColumn) {
     const linkedTable = this.tables.find((t) => (t.initialName || t.name) === column.link?.table);
     function getType() {
-      if (!linkedTable) return `(${chalk.gray.italic(column.type)})`;
-      return `(${chalk.gray.italic(column.type)} → ${chalk.gray.italic(linkedTable.name)})`;
+      if (!linkedTable) return chalk.gray.italic(column.type);
+      return `${chalk.gray.italic(column.type)} → ${chalk.gray.italic(linkedTable.name)}`;
     }
-    const type = getType();
+    const metadata = [
+      getType(),
+      column.unique ? chalk.gray.italic('unique') : '',
+      column.notNull ? chalk.gray.italic('not null') : ''
+    ]
+      .filter(Boolean)
+      .join(' ');
     if (table.deleted || column.deleted || linkedTable?.deleted)
-      return `- ${chalk.red.strikethrough(column.name)} ${type}`;
-    if (table.added || column.added) return `- ${chalk.green(column.name)} ${type}`;
+      return `- ${chalk.red.strikethrough(column.name)} (${metadata})`;
```

Comment:
If `metadata` is "" it will print:

```
- ColumnName ()
```

Reply:
But the metadata always contains at least the type, so it won't be an empty string.

Reply:
True!